macro/load_all_libs.C

```diff
@@ -14,10 +14,6 @@ void load_all_libs()
    gSystem->Load("libMFTSimulation");
    gSystem->Load("libMathUtils");
    gSystem->Load("libO2Device");
-   gSystem->Load("libQCMerger");
```

Comment:
@Barthelemy : seems these libs are no longer build. Could we also remove the old buckets in the O2Dependencies.cmake?

I believe this file should ideally be auto-generated to be always up to date (if we need the file at all).

Reply:
yes, we can remove. There will be a PR to cleanup the so called "QC" directory and to replace it with the mergers.

Reply:
The PR https://github.com/AliceO2Group/AliceO2/pull/1653 removing the QC directory was already merged, but indeed I forgot to delete the buckets.